lib/node_modules/@stdlib/array/one-to/docs/types/index.d.ts

```diff
@@ -201,7 +37,7 @@ declare function oneTo( n: number, dtype: 'generic' ): Array<number>;
 * var arr = oneTo( 2, 'float32' );
 * // returns <Float32Array>[ 1.0, 2.0 ]
 */
-declare function oneTo( n: number, dtype?: NumericDataType ): Float64Array;
+declare function oneTo<T extends keyof DataTypeMap<number> = 'float64'>( n: number, dtype?: T ): DataTypeMap<number>[T];
```

Comment:
This is a use case for `NumericAndGenericDataTypeMap`. The previous `NumericDataType` failed to capture "generic", but was trending in the right direction with only numeric dtypes being allowed.

Reply:
Changed.